kalite/topic_tools/__init__.py

```diff
@@ -156,14 +175,12 @@ def get_exercise_cache(force=False, language=settings.LANGUAGE_CODE):
                         available = True
                 exercise["all_assessment_items"] = items
             else:
-                available = os.path.isfile(TEMPLATE_FILE_PATH % exercise_template)
+                available = exercise_template in exercise_templates
 
                 # Get the language codes for exercise templates that exist
                 # Try to minimize the number of os.path.exists calls (since they're a bottleneck) by using the same
                 # precedence rules in i18n.select_best_available_languages
-                available_langs = [lang_code for lang_code in exercise_templates if language in lang_code or lang_code == settings.LANGUAGE_CODE]
-                available_langs = [lang_code for lang_code in available_langs if os.path.exists(os.path.join(exercise_root, lang_code, exercise_file))]
-                available_langs = set(["en"] + available_langs)
+                available_langs = set(["en"] + [language]*available)
```

Comment:
`list` * `bool`? I had to try it out myself. I can't seem to find anything on it in the docs, is this potentially undefined behavior?